internal/clients/team4/client.go

```diff
@@ -13,8 +13,14 @@ import (
 
 const id = shared.Team4
```

Comment:
remove

Reply:
Did it. Broke team 3 tests. This code hates me.